### DISCUSSION TITLE
little confusion or seeking more Clarification on below points with proper examples.

### DISCUSSION THREAD

**Discussion by ravipawar23** [snipe/snipe-it#12811] [General]
little confusion or seeking more Clarification on below points with proper examples.  

Asset Name ?
Asset Tags ?
Model ?
Model No ?
Location ?
Default location ?
Asset labels ?
Asset Tags ?
Barcodes ? - can we use the prefix in it ?

there is no details about Warranty start date ?, warranty starts after delivery of hardware.

some times the delivery takes more time then expected, so there should be the separate column for warranty start date.

**Comment by snipe** [comment]
I'm unclear about what you're unclear on here TBH. They're kind of self-explanatory. 

> Asset Name ?

Optional - use it or don't. Call it "ravipawar23's MacBook Pro", "Loaner Laptop", or call it nothing at all.

> Asset Tags ?

A unique identifier for the asset within the system. Asset tags are a common vernacular in IT.

> Model ?

Literally what is the mode of the asset? MacBook Pro 13"? Lenovo Yoga? 

> Model No ?

Optional, but the actual model number (if whatever you're tracking has one)

> Location ?

Where is it?

> Default location ?

Where does it live when it's not assigned? (Warehouse, inventory closet, some other branch office, etc)

> Asset labels ?

Labels that would go on the asset to identify it.

> Barcodes ? - can we use the prefix in it ?

It's an image code - how do you prefix an image?
